### PR TITLE
Adding metadata.

### PR DESCRIPTION
**Pull request by dend**

### CODE REVIEW

File: docs/languages/csharp/index.md [hunk 1]
```diff
@@ -1,4 +1,18 @@
-﻿# Learn C# # 
+---
+title: Learn C
```
Comment (mairaw) [inline docs/languages/csharp/index.md]: does this field supports the # character?

**Comment by mairaw** [comment]
thanks @dend! @richlander is it ok to merge these metadata as-is and then we do a second PR to fix author names, remove bylines, etc.? have you had a chance to read about the metadata discussion today?

File: docs/languages/csharp/index.md (hunk 1 above)
Comment (dend) [inline docs/languages/csharp/index.md]: It does, but this seems like an artifact - feel free to re-add the `#`

**Comment by mairaw** [comment]
I'm merging this in so we can get this in with no merging conflicts. I'll work on updating the authors and removing the bylines later on.

**Pull request merged**